### PR TITLE
KAH-25

### PR DESCRIPTION
**Pull request by DavidIzaac**

### CODE REVIEW

File: server/api/tests/plugins.test.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ describe('Plugins tests', () => {
         });
 
         // notice: redundant "/delete"
```
Comment (arispen) [inline server/api/tests/plugins.test.js]: you can delete this comment already :)

File: server/api/tests/plugins.test.js [hunk 2]
```diff
@@ -112,11 +112,11 @@ describe('Plugins tests', () => {
         describe(`GET /:id`, () => {
             it(`should respond with status 500 and proper error msg for invalid id`, () => {
```
Comment (arispen) [inline server/api/tests/plugins.test.js]: test description says it should respond with 500 and you respond with 404

File: server/api/tests/plugins.test.js [hunk 3]
```diff
@@ -112,11 +112,11 @@ describe('Plugins tests', () => {
...
                 return request(app)
-                    .get('/api/plugins/0')
-                    .expect(500)
-                    .then((res) => {
-                        expect(res.body.message).toBe("Cast to ObjectId failed for value \"0\" at path \"_id\" for model \"Plugin\"");
-                    });
+                    .get('/api/plugins/5d83a7dac12c6d695c3cbc66')
+                    .expect(404)
```
Comment (arispen) [inline server/api/tests/plugins.test.js]: does this change affect frontend?

Comment (ChBernat) [reply server/api/tests/plugins.test.js]: This is a change in tests only and does not affect front-end.

File: server/api/controllers/plugins.controller.js [hunk 4]
```diff
@@ -30,6 +30,9 @@ module.exports = {
         hooks.hookPre('get-plugin', req).then(() => {
             return pluginsService.getPlugin(req.params.id);
         }).then((plugins) => {
+            if (!plugins) {
+                return res.status(404).send('Plugin not found')
```
Comment (ChBernat) [inline server/api/controllers/plugins.controller.js]: How does this affect front-end?

**Pull request merged**